src/containerapp/azext_containerapp/custom.py

```diff
@@ -2064,6 +2072,105 @@ def connected_env_remove_storage(cmd, storage_name, name, resource_group_name, n
         handle_raw_exception(e)
 
 
+def setup_core_dns(cmd, distro=None, kube_config=None, kube_context=None, skip_ssl_verification=False):
```

Comment:
Seems the `distro` has not been used? Do we really need this param?
![image](https://github.com/user-attachments/assets/eb0fc6c5-e231-44fa-8974-0bfe5927dcdb)

Reply:
This parameter is used to let cli check the distro parameter and value, for now, it only have one value, but in the future, we may support more type.